packages/framework-integration-tests/integration/providers/aws/utils.ts

```diff
@@ -525,6 +519,11 @@ export async function countTableItems(tableName: string): Promise<number> {
 }
 
 export async function countEventItems(): Promise<number> {
+  // TODO: This way of counting is wrong when there are a lot of events in the event store, as COUNT will only contain
+  // the number of elements in a a result set that is lower than 1MB in size:
+  // "If the size of the Query result set is larger than 1 MB, then ScannedCount and Count
+  // will represent only a partial count of the total items.
+  // You will need to perform multiple Query operations in order to retrieve all of the results\"."
```

Comment:
DYNAMODB....

![image](https://user-images.githubusercontent.com/2271237/110357080-d1b5ba00-8032-11eb-9e51-e8ca9bfcfc14.png)

We need to start creating tests with huge loads of events. We will start finding hidden bugs.